knexfile.js

```diff
@@ -2,7 +2,7 @@ require('dotenv').config({ path: `.env.${process.env.NODE_ENV}` });
 const path = require('path');
 const expect = require('expect-runtime');
 
-const connection = process.env.DATABASE_URL_SEEDER;
+const connection = process.env.DATABASE_URL;
 
```

Comment:
We need to use DATABASE_URL_SEEDER since this is the m_{service} user which can not perform DELETE, while DATABASE_URL is just the s_{service} user, which cannot delete from the tables.

running scripts/setup-setup-dev-database-passwords.sh will set up all these env vars